### PR TITLE
wallet: remove force option from wallet.makeOpen.

### PR DESCRIPTION
**Pull request by nodech**
`force` option is leftover after refactors as far as I can see in the history.
As it is not used and no one knows why it was there, might as well clean up.

### CODE REVIEW

**Review by rithvikvibhu — APPROVED** [review]
Closes #93!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # HSD Release Notes & Changelog
 
+## v6.0.0
+
+### Wallet changes
+
+- HTTP Changes:
+  - `/wallet/:id/open` no longer accepts `force` flag. (it was not used)
+- RPC Changes:
+  - `createopen` and `sendopen` no longer accept `force` as an argument. (was not used)
```
[the remark this answers is not in the document]
Comment (nodech) [reply CHANGELOG.md:10]: It sounds too much to me. It would be good to document every change like that for the lower level consumers of the hsd, but given the ability to reorganize/refactor some code that does not break intended API that is HTTP/RPC only - gives us more flexibility.

I would consider changes to MTX/TX/Script etc primitives that are almost necessary in some front-end cases (e.g. working with ledger etc) to be part of the breaking changes. 

Given wallet is meant to be consumed via HTTP, I think documenting this as a breaking change to become more troublesome. If we were to go that way, every change to any class should be documented here as well, as more involved use cases may use those. 

So, in summary, I would stick to the current rules: document breaking API (HTTP/RPC/DNS) and MAYBE primitives.

[the remark this answers is not in the document]
Comment (rithvikvibhu) [reply CHANGELOG.md:10]: Makes sense 👍

**Comment by nodech** [comment]
Rebased
